cmd/kube-apiserver/app/testing/server_test.go

```diff
@@ -384,3 +385,65 @@ func crdExistsInDiscovery(client apiextensionsclientset.Interface, crd *apiexten
 	}
 	return false, nil
 }
+
+// TestOpenAPIPresence validates that various paths exist in the
+// complete delegation chain of the server. If the chain is not wired
+// up correctly this test should fail; this is largely a smoke test to
+// ensure that the complete delegation chain is complete.
+func TestOpenAPIPresence(t *testing.T) {
+	config, tearDown := StartTestServerOrDie(t)
+	defer tearDown()
+
+	kubeclient, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		t.Fatalf("Unexpected error: %v", err)
+	}
+
+	result := kubeclient.RESTClient().Get().AbsPath("/swagger.json").Do()
+	status := 0
+	result.StatusCode(&status)
+	if status != 200 {
+		t.Fatalf("GET /swagger.json failed: expected status=%d, got=%d", 200, status)
+	}
+
+	raw, err := result.Raw()
+	if err != nil {
+		t.Fatalf("Unexpected error: %v", err)
+	}
+
+	type openAPISchema struct {
+		Paths map[string]interface{} `json:"paths"`
+	}
+
+	var doc openAPISchema
+	err = json.Unmarshal(raw, &doc)
+	if err != nil {
+		t.Fatalf("Failed to unmarshal: %v", err)
+	}
+
+	matchedExtension := false
+	extensionsPrefix := "/apis/apiextensions.k8s.io"
```

Comment:
can you reference the GroupName in the external type packages instead of hardcoding this here? Also please add a comment that these are representatives for apiextensions-apiserver and kube-aggregator, both part of the delegation chain in kube-apiserver. So this is a smoke-test that we wired this correctly.